lm_eval/utils.py

```diff
@@ -56,7 +55,7 @@ def simple_parse_args_string(args_string):
     if not args_string:
         return {}
     arg_list = args_string.split(",")
```

Comment:
would you consider `arg_list = [arg for arg in args_string.split(",") if arg]`. I always add a trailing comma, keep forgetting that doesn't work and spend ages debugging

Reply:
Of course! Made this change.